Sources/Build/describe().swift

```diff
@@ -128,6 +128,11 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
     #if os(Linux)
          args += ["-fPIC"]
     #endif
+    #if os(OSX)
+        if let sysroot = Toolchain.sysroot {
```

Comment:
@ddunbar I just checked the source code. The end of the file Sources/Utility/Toolchain.swift is as follows:

#if os(OSX)
extension Toolchain {
    public static let sysroot = getenv("SYSROOT") ?? (try? POSIX.popen(["xcrun", "--sdk", "macosx", "--show-sdk-path"]))?.chuzzle()
    public static let platformPath = (try? POSIX.popen(["xcrun", "--sdk", "macosx", "--show-sdk-platform-path"]))?.chuzzle()
}
#endif

Thus Toolchain.sysroot is only defined on OSX and a short test of trying to use Toolchain.sysroot on Linux failed to compile.